### DISCUSSION TITLE
How to add custom fonts to micropython.zip?

### DISCUSSION THREAD

**Discussion by Eloitor** [wasp-os/wasp-os#435] [Q&A]
I added new fonts to `wasp/fonts` using `font_to_py.py` and I included them into `wasp/fonts/__init__`.

Using `make sim`, works fine, and I can use them.
But when building from source using:

```
make submodules
make softdevice
make -j `nproc` BOARD=pinetime all
```

The generated micropython.zip does not work in the PineTime... What am I missing?

**Comment by fgaz** [comment]
Does it work if you `make clean` first?

**Reply by Eloitor** [reply]
It doesn't. I removed everything and cloned it again from my fork. The build with this commit does not work: https://codeberg.org/Eloitor/wasp-os/commit/ead84bf93f8f71e76ef3829e929e1784a81046a8


I'm only adding the fonts... I don't include any app that uses it to the micropython.zip since I add the apps/*.py files manually.

**Comment by Eloitor** [comment]
I did some progress... I figured out I should add them to `wasp/boards/manifest_240x240.py` but the micropython.zip ins't working yet...

**Comment by Eloitor** [comment]
This was the solution: 
- Add new fonts to wasp/fonts using font_to_py.py
- Include them into wasp/fonts/__init__ and wasp/boards/manifest_240x240.py
